Source/JavaScriptCore/jit/Width.h

```diff
@@ -93,6 +93,7 @@ inline constexpr uint64_t mask(Width width)
     case Width64:
         return 0xffffffffffffffffllu;
     case Width128:
+    default:
```

Comment:
This doesn't make sense though—the `case`s are exhaustive so there can be no `default` path.

Reply:
Compiler disagrees:

```
In file included from /app/webkit/Source/JavaScriptCore/b3/B3Type.h:32,
                 from /app/webkit/Source/JavaScriptCore/b3/B3Bank.h:30,
                 from /app/webkit/Source/JavaScriptCore/b3/B3Value.h:30,
                 from /app/webkit/Source/JavaScriptCore/b3/B3FixSSA.h:30,
                 from /app/webkit/Source/JavaScriptCore/b3/B3Generate.cpp:37,
                 from /app/webkit/WebKitBuild/Debug/JavaScriptCore/DerivedSources/unified-sources/UnifiedSource-23a5fd0e-5.cpp:1:
/app/webkit/Source/JavaScriptCore/jit/Width.h: In function ‘constexpr uint64_t JSC::mask(Width)’:
/app/webkit/Source/JavaScriptCore/jit/Width.h:99:1: error: control reaches end of non-void function [-Werror=return-type]
```

Reply:
No, I understand that the compiler is unhappy, but it's not because the `switch` isn't exhaustive; it's that the compiler is unwilling to count `case`s when checking whether the function as a whole has a consistent return type.

_Adding_ a `default` is incorrect for any human reader because there are only five possible paths, but we can _replace_ the last one with a `default`.